resources/options_lookup.tsv

```diff
@@ -938,17 +940,17 @@ Cooling Setpoint Offset Magnitude	2F	ResidentialHVACCoolingSetpoints	weekday_off
-Cooling Setpoint Offset Period	Day Setup +1h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup +2h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup +3h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup +4h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup +5h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -1h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -2h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -3h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -4h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -5h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0"																																											
+Cooling Setpoint Offset Period	Day Setup	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0"																																											

```

Comment:
@afontani Why this change but not for Night Setup?

Reply:
@joseph-robertson It is assumed that a day setback is usually applied during unoccupied times, which we can't assume during weekends. However, if there is a night setback that would be the desired temperature while asleep, which probably would not be affected by weekday/weekend days.